### PR TITLE
Additional device check for Ipad Pro on Safari

### PR DESCRIPTION
**Pull request by Drkjr92**
**Description:**
Issue raised in [#5494](https://github.com/aframevr/aframe/issues/5494). 

The issue: iPadOS User Agent in Safari is same as on MacOS Catalina.

User case:
> There are some use cases in which it is necessary to know whether it is a mobile device, e.g. a help dialogue that describes either touch or mouse/keyboard gestures.

**Changes proposed:**
-  In `isTablet()` we want to extend the ipad check for those special cases where the user agent from the Ipad is labeled "MacOS"

Test also added and Results are shown below:

image

### CODE REVIEW

File: src/utils/device.js [hunk 1]
```diff
@@ -123,10 +123,29 @@ module.exports.isMobile = isMobile;
  */
 function isTablet (mockUserAgent) {
   var userAgent = mockUserAgent || window.navigator.userAgent;
-  return /ipad|Nexus (7|9)|xoom|sch-i800|playbook|tablet|kindle/i.test(userAgent);
+
+  var _isTablet = /ipad|Nexus (7|9)|xoom|sch-i800|playbook|tablet|kindle/i.test(userAgent);
```
Comment (dmarcos) [inline src/utils/device.js:127]: we don't use variable names with underscores. I also see the regular expression is checking for ipad. needed?

Comment (dmarcos) [reply src/utils/device.js:127]: thanks

Comment (dmarcos) [reply src/utils/device.js:127]: also wonder if this regex for tablet still provide good coverage. I see device versions hardcoded (e.g `Nexus 7|9`).

**Comment by dmarcos** [comment]
Thanks so much! Congrats on your first contribution

**Comment by Drkjr92** [comment]
Thank you!!

**Pull request merged**